nservicebus/containers/autofac.md

```diff
@@ -20,6 +20,8 @@ snippet:Autofac
 
 snippet:Autofac_Existing
 
+NOTE: The registrations from the container  passed in to the `ExistingLifetimeScope` method will be copied into a new container. This means that the instance of the container passed in will not be altered and none of the internal NServiceBus registrations will appear in the original container.
```

Comment:
Dunno if it matters, but there is an extra space between "container" and "passed".